connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/ClusterConfigState.java

```diff
@@ -86,6 +114,20 @@ public long offset() {
     }
 
     /**
+     * Get all task configs for a connector.
+     * @param connector name of the connector
+     * @return a map from the task id to its configuration
+     */
+    public Map<ConnectorTaskId, Map<String, String>> allTaskConfigs(String connector) {
```

Comment:
Yeah, actually I don't know why we don't use a a single `Map<String, ConnectorState>`, where the object contains all the task configs and connector state. This is how StandaloneHerder was organized before (and which I copied in MemoryConfigBackingStore).